Documented debug_traceCall (#1439)
hyperledger/besu-docs: docs/public-networks/reference/api/index.md

# JSON result

```json
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": [
    {
      "gas": 21000,
      "failed": false,
      "returnValue": "",
      "structLogs": [
        {
          "pc": 0,
          "op": "STOP",
          "gas": 0,
          "gasCost": 0,
          "depth": 1,
          "stack": [],
          "memory": [],
          "storage": null,
          "reason": null
        }
      ]
    }
  ]
}
```
<!--/tabs-->

### `debug_traceCall`

> Returns a detailed trace of the executed opcodes for a specific transaction.

Reviewer: ```suggestion
Performs an eth_call within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.
```

#### Parameters

> - `requestContext`: _object_ - JSON-RPC request context with essential fields like `from`, `to`, `gas`, `gasPrice`, `value`, `data`, `nonce`.

Reviewer: ```suggestion
`call`: _object_ - [transaction call object](objects.md#transaction-call-object)

`blockNumber` or `blockHash`: _string_ - hexadecimal or decimal integer representing a block number, block hash, or one of the string tags `latest`, `earliest`, `pending`, `finalized`, or `safe`, as described in [block parameter](../../how-to/use-besu-api/json-rpc.md#block-parameter)
```

# JSON result

```json
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": [
    {
      "gas": 21000,
      "failed": false,
      "returnValue": "",
      "structLogs": [
        {
          "pc": 0,
          "op": "STOP",
          "gas": 0,
          "gasCost": 0,
          "depth": 1,
          "stack": [],
          "memory": [],
          "storage": null,
          "reason": null
        }
      ]
    }
  ]
}
```
<!--/tabs-->

### `debug_traceCall`

> Performs an eth_call within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.

Reviewer 2: ```suggestion
Performs an [`eth_call`](#eth_call) within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.
```
